ethereum/core/src/test/java/tech/pegasys/pantheon/ethereum/privacy/PrivateTransactionTest.java

```diff
@@ -39,58 +39,53 @@
           + "601b4ab949f53faa07bd2c804";
 
   private static final String VALID_PRIVATE_TRANSACTION_RLP =
-      "0xf90113800182520894095e7baea6a6c7c4c2dfeb977efac326af552d87"
-          + "a0ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
-          + "ffff801ba048b55bfa915ac795c431978d8a6a992b628d557da5ff759b307d"
-          + "495a36649353a01fffd310ac743f371de3b9f7f9cb56c0b28ad43601b4ab94"
-          + "9f53faa07bd2c804ac41316156744d784c4355486d425648586f5a7a7a4267"
-          + "5062572f776a3561784470573958386c393153476f3df85aac41316156744d"
-          + "784c4355486d425648586f5a7a7a42675062572f776a356178447057395838"
-          + "6c393153476f3dac4b6f32625671442b6e4e6c4e594c35454537793349644f"
-          + "6e766966746a69697a706a52742b4854754642733d8a726573747269637465"
-          + "64";
+      "0xf8ef800182520894095e7baea6a6c7c4c2dfeb977efac326af552d87a0f"
```

Comment:
Test vectors have changed here because enclave keys encoded correctly as 32 byte hex rather than 44 byte UTF_8 representation of base64